session.go

```diff
@@ -102,6 +103,12 @@ func (s *Session) open() error {
 	}
 
 	header := http.Header{}
+	if strings.TrimSpace(s.originHeader) != "" {
+		_, err := url.ParseRequestURI(s.originHeader)
+		if err != nil {
```

Comment:
I just noticed that this should be

`if err = nil` instead of `if nerr != nil` for the header to be set. Should I create another PR or will you change this?

Reply:
Ah, yeah I should have caught this. I can push the update. Thanks for letting me know.